src/objective/multiclass_objective.hpp

```diff
@@ -267,6 +267,8 @@ class MulticlassOVA: public ObjectiveFunction {
   }
 
  private:
+  MulticlassOVA() {}
```

Comment:
How is this related to the other changes in this PR? I don't see this method called anywhere in the diff of this PR.

Reply:
This is for subsequent PR where CUDA objectives will be used, and they will inherit from these CPU objectives. I can remove this until the CUDA objective PR.